napari/layers/utils/text_manager.py

```diff
@@ -107,19 +118,48 @@ def __init__(
         if text is not None:
             _warn_about_deprecated_text_parameter()
             kwargs['string'] = text
+        if 'color' in kwargs:
+            _warn_about_deprecated_color_field()
+            color = kwargs.pop('color')
+            if 'face_color' not in kwargs:
+                kwargs['face_color'] = ConstantColorEncoding(constant=color)
         super().__init__(**kwargs)
         self.events.add(values=Event)
-        self.string._apply(features)
+        self.events.add(
+            color=WarningEmitter(
+                trans._(
+                    'color is a deprecated event. Use face_color instead.'
+                ),
+                type='color',
+            )
+        )
+        self.apply(features)
 
     @property
     def values(self):
         return self.string._values
 
+    @property
+    def color(self) -> np.ndarray:
+        if isinstance(self.face_color, ConstantColorEncoding):
+            _warn_about_deprecated_color_field()
+            return self.face_color.constant
+        raise NotImplementedError(
+            trans._(
+                'color is deprecated and only is only defined when face_color is a constant color encoding. Use face_color instead.'
+            )
+        )
+
     def __setattr__(self, key, value):
         if key == 'values':
             self.string = value
+        elif key == 'color':
+            _warn_about_deprecated_color_field()
```

Comment:
would be nice to send a message detailing the equivalent setter for `face_color` e.g. `manager.color = cyan` -> `manager.face_color = {'constant': cyan}`

Reply:
Great idea! I'm going to hold off on making the change until we're more certain on introducing `face_color` and the new coercion behavior.